files/en-us/mozilla/firefox/releases/93/index.html

```diff
@@ -85,6 +85,10 @@ <h4 id="removals_wasm">Removals</h4>
 
 <h3 id="webdriver_conformance_marionette">WebDriver conformance (Marionette)</h3>
 
+<ul>
+  <li>Fixed a bug, which caused the `WebDriver:Print` to fail for large documents ({{bug 1721982}}).</li>
```

Comment:
nit: Either remove `the` before `WebDriver:Print` or add `command`? 
```suggestion
  <li>Fixed a bug, which caused the command `WebDriver:Print` to fail for large documents ({{bug 1721982}}).</li>
```

Reply:
Ah, true, thanks for catching that Julian